### PR TITLE
New package: rapidjson-1.1.0

### PR DESCRIPTION
**Pull request by egorenar**

### CODE REVIEW

File: srcpkgs/rapidjson/template [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Template build file for 'rapidjson'.
+pkgname=rapidjson
+version=1.1.0
+revision=1
+nocross=yes
+build_style=cmake
+configure_args="-DCMAKE_INSTALL_PREFIX=/usr"
+make_build_args="VERBOSE=1"
```
Comment (the-maldridge) [inline srcpkgs/rapidjson/template]: Is this necessary to get a build out?  I'd rather this not be here in the merged template

File: srcpkgs/rapidjson/template [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Template build file for 'rapidjson'.
+pkgname=rapidjson
+version=1.1.0
+revision=1
+nocross=yes
```
Comment (the-maldridge) [inline srcpkgs/rapidjson/template]: This should have a comment or link to a build log explaining why this is nocross.

Comment (Vaelatern) [reply srcpkgs/rapidjson/template]: Indeed, `nocross="The Fizz is broken when Baz happens"` would (except being nonsense) be a good way to do this.

File: srcpkgs/rapidjson/template [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+build_style=cmake
+configure_args="-DCMAKE_INSTALL_PREFIX=/usr"
```
Comment (Vaelatern) [inline srcpkgs/rapidjson/template]: This is already passed as needed. Please remove this line.

File: srcpkgs/rapidjson/template (hunk 2 above)
Comment (egorenar) [reply srcpkgs/rapidjson/template]: removed nocross

File: srcpkgs/rapidjson/template (hunk 1 above)
Comment (egorenar) [reply srcpkgs/rapidjson/template]: done

File: srcpkgs/rapidjson/template (hunk 3 above)
Comment (egorenar) [reply srcpkgs/rapidjson/template]: done

**Comment by egorenar** [comment]
i addressed all raised issues, please check again. Thanks.

**Comment by Vaelatern** [comment]
In this case, it seems the main package is only documentation without the contents of the `-devel` package. Was I just not reading carefully?

If the main package has no use without the -devel package, and there is indeed no `.so` built for this library, then this package needs to be `noarch=yes` and the `-devel` package would need to be removed in favor of all files being in the main package.

**Comment by egorenar** [comment]
you are right, this library is a header-only one.

**Pull request merged**